src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LambdaHandler.java

```diff
@@ -59,6 +73,7 @@ protected IndentLevel getIndentImpl() {
 
         DetailAST parent = getMainAst().getParent();
         if (getParent() instanceof NewHandler) {
+            isLambdaCorrectlyIndented = false;
```

Comment:
This new field isn't final after it is first assigned in the constructor. That means its possible the value can flip after the constructor, and it looks like it can be flipped in multiple areas.

Is it possible usage of this field in one method can go down the `true` path and then another method could use it as the `false` path, and then re-calling the old method will then go down the `false` path the 2nd time through?
This is possible it would lead to some randomness issues in the check throughout a run.

Reply:
@rnveach ,
The value of this variable is used only **once** for suggesting the children's expected indent at line **55**
This method is generally executed after running all the methods regarding checking the Lambda's indent.
The main purpose of this variable is checking if there are indentation error logs for Lamda expression.

Reply:
I looked into my concerns with a custom branch and regression. See https://github.com/checkstyle/checkstyle/pull/8719#issuecomment-692380080 .

No exceptions were thrown that would mean we assigning the field 2 different values. So that alleviates my main concern.